test/api/v01/request_helper.rb

```diff
@@ -118,7 +120,7 @@ def self.solve_asynchronously
   ensure
     Resque.inline = old_resque_inline
     OptimizerWrapper.config[:solve][:synchronously] = old_config_solve_synchronously
-    if pgid_worker
+    if options[:start_worker]
```

Comment:
we are in test mode, if we cannot spawn worker, do you prefer there is crash to get info for why process has not been spawned or do you prefer to have something like "silent death" (hard to analyse)?

Reply:
It is easier with `&& pid_worker` I think. 

Because without it, instead of the test breaking in the expected way, we get a raise from `worker_pids = 'ps -o pgid,pid | grep #{pgid_worker}'.split(/\n/)` which "hides" the broken test. 

With it, if we couldn't launch the worker, this means there is a raise in line `Process.spawn` and if we do not "create" a new raise inside this `if` condition, then the test will correctly point to `Process.spawn` line.